runtime-config-linux.md

```diff
@@ -17,7 +17,7 @@ The following parameters can be specified to setup namespaces:
     * **`uts`** the container will be able to have its own hostname and domain name
     * **`user`** the container will be able to remap user and group IDs from the host to local users and groups within the container
 
-* **`path`** *(string, optional)* - path to namespace file
+* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
```

Comment:
On Fri, Dec 18, 2015 at 01:01:41PM -0800, Mrunal Patel wrote:
> > -* **`path`** *(string, optional)* - path to namespace file
> > +* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
> 
>  I meant that we can close this PR.

I tripped over this problem in ccons, so I think some sort of
clarification is useful.  I think most folks will want the
interpretation I'm proposing, but I don't see a fundamental problem
with an interpretation like:

  The container proccess creates/joins namespaces in the order listed
  in the config, and paths are in the current mount namespace.

Which would allow you to do things like:

  "namespaces": [
    {
      "type": "mount"
      "path": "/mnt/test/mnt",
    },
    {
      "type": "net",
      "type": "/proc/1/ns/net",
    }
    {
      "type": "mount"
    }
  ]

For:

1. Join the mount namespace at /mnt/test/mnt (which happens to hold a
   /proc for a different PID namespace).
2. Join the network namespace for that different PID namespace's init
   process.
3. Create a new mount namespace for the container.

I think we do want some kind of wording to distinguish between these
two interpretations.  And we probably want wording about whether the
order in the namespaces array is significant (along the lines of #142
and #157 if it is, and probably convert the array to an object keyed
by name if it is not).